tables/servers.txt

```diff
@@ -94,7 +94,7 @@ serverType iRO
 serverEncoding Western
 patchserver ropatch1.gravityus.com
 patchpath /patch02
-addTableFolders iRO
+addTableFolders iRO/Renewal;iRO
```

Comment:
iRO RE and PRE 
Use with the same client?

Reply:
@sctnightcore 
i dont know, maybe not, because classic doesn't use shuffle or sync

Reply:
@allanon  @alisonrag 
I'm sure it will use the same client.